Header header

geometry_msgs/Vector3 translation			# body xyz coordinate (inertial frame)
geometry_msgs/Vector3 linear_velocity		# body xyz velocity (inertial frame)
geometry_msgs/Vector3 linear_acceleration	# body xyz acceleration (inertial frame)

geometry_msgs/Quaternion orientation		# body orientation (transforms body to inertial frame)
geometry_msgs/Vector3 angular_velocity		# body angluar velocity (inertial frame)
geometry_msgs/Vector3 angular_acceleration	# body angular acceleration (inertial frame)